conways-game-of-life/src/scripts/grid.js

```diff
@@ -0,0 +1,95 @@
+const gridLife = gridArray => {
+  const grid = gridArray;
+  const width = gridArray.length;
+  const height = gridArray[0].length;
+
+  const validateCoords = (x, y) => {
+    let xValid = x;
+    let yValid = y;
```

Comment:
If you use the arguments of xValid and yValid instead of x and y in validateCoords then you wouldn't have to reassign your arguments to different variable names

Reply:
I don't fully understand what you mean in this comment but what I was avoiding to do is to mutate the x and y variables